get_github_coderabbit_review/coderabbit_fetcher/analyzers/resolution_detector.py

```
@@ -0,0 +1,421 @@
+            r'addressed',
+        ]
+
+        for pattern in definitive_patterns:
+            if re.search(pattern, latest_marker.marker_text, re.IGNORECASE):
+                return ResolutionStatus.RESOLVED
+
+        # 複数のマーカーがある場合は解決済みと判定
+        if len(resolution_markers) >= 2:
+            return ResolutionStatus.RESOLVED
+
+        # 単一マーカーの場合は未解決として扱う（解決の証拠が不十分）
```

Ruff (0.13.1)

Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)